var/spack/repos/builtin/packages/py-pyside/package.py

```diff
@@ -93,5 +99,8 @@ def patch(self):
             "setup.py",
         )
 
+        # remove check for python version
+        filter_file(r"^check_allowed_python_version()", "", "setup.py")
```

Comment:
```suggestion
        filter_file("^check_allowed_python_version()", "", "setup.py")
```
You're not escaping anything